docs/en/ingest-management/elastic-agent/elastic-agent-unprivileged-mode.asciidoc

```diff
@@ -36,50 +36,89 @@ Doing so may result in <<agent-sudo-error,an error>> due to the agent not having
+|Give read permission to the `elastic-agent` group for the `/var/log/syslog` and `/var/log/auth.log` files to fix the missing visualizations.
 
-|<<elastic-agent-inspect-command,`inspect` command>>
-|Returns only...
-|`Exmaple error message...`
+|Run {agent} and access the {agent} dashboards
+|On the `[Elastic Agent] Agents info` dashboard, visualizations including `Most Active Agents` and `Integrations per Agent` are missing data.
+|Giving read permission to the `elastic-agent` group for the `/var/log/system.log` file will partially fix the visualizations, but errors may still occur because the `elastic-agent-user` does not have read access to files in the `/run/user/1000/` directory.
+// It'd be nice if we can expand on this, even if just to say why that read access can't be given.
+
+|Run {agent} and access the {agent} dashboards
+|On the `[Elastic Agent] Integrations` dashboard, visualizations including `Integration Errors Table`, `Events per integration` and `Integration Errors` are missing data.
+|Give read permission to the `elastic-agent` group for the `/var/log/system.log` file to fix the missing visualizations.
 
-|<<elastic-agent-status-command,`status` command>>
-|Returns only...
-|`Example error message...`
 |===
 
 .Windows
 [options,header]
 |===
-|Command |Behavior in unprivileged mode |Error message
+|Action |Behavior in unprivileged mode |Resolution
+
+|Run {agent} with the System integration
+|Log file error: `failed to open Windows Event Log channel "Security": Access is denied`
+|Add the `elastic-agent` user to the `Event Log Users` group to fix this error.
+
+|Run {agent} with the System integration
+|Log file error: `cannot open new key in the registry in order to enable the performance counters: Access is denied`
+|Update the permissions for the `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PartMgr` registry to fix this error.
 
-|<<elastic-agent-inspect-command,`inspect` command>>
-|Returns only...
-|`Exmaple error message...`
+|Run {agent} with the System integration
+|Most of the System and {agent} dashboard visualizations are missing all data.
+|Add the `elastic-agent` user to the `Event Log Users` group and update the permissions for the `HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PartMgr` registry to fix the missing visualizations.
+
+Note that the `elastic-user-agent` user may still not have access to all processes, so the lists in the `Top processes by CPU usage` and `Top processes by memory usage` visualizations may be incomplete. 
+
+|Run {agent} with the System integration
+|On the `[Metrics System] Host overview` dashboard, the `Disk usage` visualizations are missing data. 
+|This occurs because direct access to the disk or a volume is restricted and not available to users without administrative privileges. Refer to link:https://learn.microsoft.com/en-us/windows/win32/secbp/running-with-special-privileges[Running with Special Privileges] in the Microsoft documentation for details.
```

Comment:
I don't know exact which groups provide what but datadogs agent automatically adds the user to these groups allowing them access to a lot of information on Windows:

https://docs.datadoghq.com/agent/guide/windows-agent-ddagent-user/

Probably adding the `elastic-agent-user` to some of those groups might fix some issues here.

Reply:
Here's what I've added:

![Screenshot 2024-06-24 at 10 16 04 AM](https://github.com/elastic/ingest-docs/assets/41695641/01eef7df-7a4b-44cc-a11a-9f73dfa53918)

@ycombinator, @VihasMakwana If possible it'd be really nice to test the newly added resolution.

Reply:
@VihasMakwana would you mind testing the resolution David added (see screenshot in previous comment) and posting your findings here?  Thanks.

Reply:
@blakerouse @ycombinator it turns out that datadog is using the Win32 PDH functions.
more about it here https://learn.microsoft.com/en-us/windows/win32/perfctrs/using-the-pdh-functions-to-consume-counter-data

----

@blakerouse even if i add the user to "Performance Monitor Group", it's not getting past [this section](https://github.com/elastic/elastic-agent-system-metrics/blob/main/metric/system/diskio/diskstat_windows_helper.go#L98-L104) (i.e. opening a volume) and it's throwing the error.

Reply:
@VihasMakwana dang it, was worth a shot.

Reply:
Ah, too bad. Thanks a lot for testing it @VihasMakwana!

I've removed this:
>Adding the `elastic-agent-user` user to the `Performance Monitor Users` group, `Event Log Readers` group, and `Performance Log Users` group should resolve the missing data in the visualations.